### DISCUSSION TITLE
Adding local .tgz file

### DISCUSSION THREAD

**Discussion by antran-mwb** [yarnpkg/berry#4032] [Q&A] [answered]
In yarn 1 we were able to add local dependencies by running the following:
`yarn add file:<path_to_file>.tgz`

This command doesn't seem to work anymore in Yarn 2+.  Is there a new syntax for this?

**Comment by arcanis** [comment, marked as answer]
You need the package name in the cli. ie `yarn add foo@file:...`

**Reply by BigBallard** [reply]
This does not seem to be documented in the new v.2 documentation for `yarn add`.
